lib/geo/wkb.ex

```diff
@@ -22,24 +22,79 @@ defmodule Geo.WKB do
   what the byte order will be.
   """
   @spec encode!(Geo.geometry(), Geo.endian()) :: binary | no_return
-  defdelegate encode!(geom, endian \\ :xdr), to: Encoder
+  def encode!(geom, endian \\ :xdr) do
+    geom |> Encoder.encode!(endian) |> Base.encode16()
+  end
 
   @doc """
   Takes a Geometry and returns a WKB string. The endian decides
   what the byte order will be.
   """
   @spec encode(binary, Geo.endian()) :: {:ok, binary} | {:error, Exception.t()}
-  defdelegate encode(geom, endian \\ :xdr), to: Encoder
+  def encode(geom, endian \\ :xdr) do
+    {:ok, geom |> Encoder.encode!(endian) |> Base.encode16()}
+  rescue
+    exception ->
+      {:error, exception}
+  end
+
+  @doc """
+  Takes a Geometry and returns a WKB string as iodata. The endian decides
+  what the byte order will be.
+  """
+  @spec encode_iodata!(Geo.geometry(), Geo.endian()) :: binary | no_return
```

Comment:
what do you think about calling it `encode_to_iodata!`?

it would nicely match [`Jason.encode_to_iodata!/2`](https://hexdocs.pm/jason/Jason.html#encode_to_iodata!/2) and [`NimbleCSV.dump_to_iodata/1`](https://hexdocs.pm/nimble_csv/NimbleCSV.html#c:dump_to_iodata/1)

In fact, following NimbleCSV, perhaps it should be called `encode_to_iodata` (no `!`) and the idea of not wrapping it in an ok/error tuple is we don't ever expect it to fail - if it does it means the user called in incorrectly, they passed an invalid struct or an invalid endianness, which would return in a FunctionClauseError and such and that is a reasonable outcome.

Reply:
Ah yeah that makes sense on both accounts.  I can update it to `encode_to_iodata` and get rid of the version that returns an ok/error tuple